src/App.js

```diff
@@ -8,9 +8,9 @@ import Privacy from './components/About/Privacy';
 import Tearms from './components/About/Terms';
 import Shop from './components/Shop/Shop';
 import Login from './components/Login/Login';
-import Signup from './components/Signup/Signup';
+import Register from './components/Signup/Signup';
```

Comment:
Replace ALL relative paths in ALL files. 
Remove everything related to login and registration.